hackathon_site/dashboard/frontend/src/components/team/TeamCardAdmin/TeamCardAdmin.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { render } from "testing/utils";
+import { teamsList } from "testing/mockData";
+
+import Teams from "pages/Teams/Teams";
+import TeamCardAdmin from "components/team/TeamCardAdmin/TeamCardAdmin";
+
+test("renders team card without crashing", () => {
+    const teamCode = teamsList[0].TeamName;
+    const members = teamsList[0].Members;
+
+    const { getByText } = render(
+        <TeamCardAdmin teamCode={teamCode} members={members} />
```

Comment:
you don't need to change this but something cool in react is you can use the spread operator to do this:
`<TeamCardAdmin {...{ teamName, members }} />` and its equivalent to what you have now

Reply:
done!